src/NLog/Targets/ConsoleTarget.cs

```diff
@@ -147,6 +164,31 @@ protected override void CloseTarget()
             base.CloseTarget();
         }
 
+        private static bool IsConsoleAvailable()
```

Comment:
Well I noticed the duplicate code-smell regarding the PauseLogging-property, so I thought you wanted the copy-paste-code for these two classes.